src/Nowruz/modules/userProfile/components/about/education/useEducation.tsx

```diff
@@ -63,6 +62,10 @@ export const useEducation = () => {
     dispatch(setIdentityType('users'));
   };
 
+  const handleOpenRequestCertificate = () => {
```

Comment:
what is this function for?

Reply:
This is going to be implemented in another ticket. I just created the name to know the place for implementation later